docs/CNFCertification-1.1-beta.md

```diff
@@ -0,0 +1,439 @@
+# CNF Certification List of Tests - v1.0-beta
+
+## Summary
+This document provides a summary of the tests included in the CNF Certification. Each test lists a general overview of what the test does, a link to the test code for that test, and links to additional information when relevant/available.
+
+To learn how to run these tests, see the "Instructions." For further details see the [USAGE guide](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md)
+
+To learn why these tests were written, see the [RATIONALE.md](https://github.com/cncf/cnf-testsuite/blob/main/RATIONALE.md)
+
+### Types of Tests (Currently 57 total for the Certification)
```

Comment:
@agentpoyo has the number of tests changed after removing 2? were any other tests added? Please review this section for accuracy

Reply:
The count in the list is now 55. I think I saw 54 tests when I ran `cert` to validate https://github.com/cncf/cnf-certification/issues/38 so I'll get confirmation on actual number.

Reply:
There are 56 tests. I think we might of miscounted to get to 57 in the original doc. I'm going to confirm that there is indeed 56 labeled as `cert` in the code now. @lixuna

Reply:
https://github.com/cncf/cnf-certification/issues/38#issuecomment-1352047218

Reply:
According to the v1 list of tests spreadsheet https://docs.google.com/spreadsheets/d/1xOxrXzexUaV-hLnPCT7NgcF6pWVWuVUrsDMXFIc8XXc/edit#gid=836418693

We did not have rolling_update as a v1 test, it's not marked as a `cert` test currently and does not run, but it is listed in the List of Tests so I will get that removed.